neqo-transport/src/cc/classic_cc.rs

```diff
@@ -164,7 +164,7 @@ impl<T: WindowAdjustment> CongestionControl for ClassicCongestionControl<T> {
         let mut is_app_limited = true;
         let mut new_acked = 0;
         for pkt in acked_pkts {
-            qinfo!(
```

Comment:
Please see this [python script](https://github.com/mozilla/neqo/blob/main/test/mozlog-neqo-cwnd.py) that we use to generate some graphs from the `qinfo` logs.
Can we not change this to `qdebug!`?

Reply:
How about running [`upload_test.sh`](https://github.com/mozilla/neqo/blob/main/test/upload_test.sh) with `RUST_LOG=neqo_transport::cc=debug` instead?

That way [`mozlog-neqo-cwnd.py`](https://github.com/mozilla/neqo/blob/main/test/mozlog-neqo-cwnd.py) still functions and plain `neqo-client` and `neqo-server` invocations aren't noisy.

What do you think @KershawChang?

If that sounds good, I can extend `upload_test.sh`.

Reply:
@KershawChang is https://github.com/mozilla/neqo/pull/1692/commits/b090a931e6c2d791f3f7843efde5c48d8fa43905 an option for the above concern?

Reply:
Yes, I think it's fine.
Thanks.